tutorials/models/1_gnn/1_gcn.py

```diff
@@ -1,41 +1,42 @@
 """
 .. _model-gcn:
 
-Graph Convolutional Network
+Graph convolutional network
 ====================================
 
 **Author:** `Qi Huang <https://github.com/HQ01>`_, `Minjie Wang  <https://jermainewang.github.io/>`_,
 Yu Gai, Quan Gan, Zheng Zhang
 
-This is a gentle introduction of using DGL to implement Graph Convolutional
-Networks (Kipf & Welling et al., `Semi-Supervised Classification with Graph
-Convolutional Networks <https://arxiv.org/pdf/1609.02907.pdf>`_). We build upon
-the :doc:`earlier tutorial <../../basics/3_pagerank>` on DGLGraph and demonstrate
-how DGL combines graph with deep neural network and learn structural representations.
+This introduction to using Deep Graph Library (DGL) to implement graph convolutional neural 
+networks (GCN), builds upon the :doc:`PageRank with DGL Message Passing <../../basics/3_pagerank>` tutorial. 
+Here you learn how DGL combines graph with a deep neural network and learn structural representations.

```

Comment:
learn -> learns

Reply:
In the last sentence, the subject is singular (you) and so the verb form matches (learn).